Description: Don't clobber CFLAGS set by main debian/rules file (allows hardening flags to pass through)
Author: Hamish Bowman <hamish_b yahoo com>
Index: opencpn/plugins/grib_pi/CMakeLists.txt
===================================================================
--- opencpn/plugins/grib_pi/CMakeLists.txt	2011-08-02 23:30:34.000000000 +1200
+++ opencpn/plugins/grib_pi/CMakeLists.txt	2012-06-16 15:28:25.621929917 +1200
@@ -32,7 +32,7 @@
  ADD_DEFINITIONS( "-Wall -O3 -fexceptions -fvisibility=hidden" )
 
  IF(NOT APPLE)
-  SET(CMAKE_SHARED_LINKER_FLAGS "-Wl,-Bsymbolic")
+  SET(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -Wl,-Bsymbolic")
  ELSE(NOT APPLE)
   SET(CMAKE_SHARED_LINKER_FLAGS "-Wl -undefined dynamic_lookup")
  ENDIF(NOT APPLE)
Index: opencpn/plugins/dashboard_pi/CMakeLists.txt
===================================================================
--- opencpn/plugins/dashboard_pi/CMakeLists.txt	2011-08-02 23:30:34.000000000 +1200
+++ opencpn/plugins/dashboard_pi/CMakeLists.txt	2012-06-16 15:27:48.533933730 +1200
@@ -33,7 +33,7 @@
  ADD_DEFINITIONS( "-Wall -O3 -fexceptions -fvisibility=hidden" )
 
  IF(NOT APPLE)
-  SET(CMAKE_SHARED_LINKER_FLAGS "-Wl,-Bsymbolic")
+  SET(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -Wl,-Bsymbolic")
  ELSE(NOT APPLE)
   SET(CMAKE_SHARED_LINKER_FLAGS "-Wl -undefined dynamic_lookup")
  ENDIF(NOT APPLE)
Index: opencpn/plugins/demo_pi_sample/CMakeLists.txt
===================================================================
--- opencpn/plugins/demo_pi_sample/CMakeLists.txt	2011-08-02 23:30:34.000000000 +1200
+++ opencpn/plugins/demo_pi_sample/CMakeLists.txt	2012-06-16 15:30:12.885859634 +1200
@@ -43,7 +43,7 @@
  ADD_DEFINITIONS( "-Wall -g -fexceptions -fvisibility=hidden" )
 
  IF(NOT APPLE)
-  SET(CMAKE_SHARED_LINKER_FLAGS "-Wl,-Bsymbolic")
+  SET(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -Wl,-Bsymbolic")
  ELSE(NOT APPLE)
   SET(CMAKE_SHARED_LINKER_FLAGS "-Wl -undefined dynamic_lookup")
  ENDIF(NOT APPLE)
